packages/wallets/src/features/accounts/modules/DocumentService/components/VerifyPersonalDetails/VerifyPersonalDetails.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { Field, useFormikContext } from 'formik';
+import moment from 'moment';
+import { TSocketError } from '@deriv/api-v2/types';
+import { DerivLightNameDobPoiIcon } from '@deriv/quill-icons';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { DatePicker, FormField, InlineMessage, WalletText } from '../../../../../../components';
+import { VerifyPersonalDetailsErrorMessage } from './components';
+import { TVerifyPersonalDetailsValues } from './types';
+import {
+    dateOfBirthValidator,
+    firstNameValidator,
+    lastNameValidator,
+    validateArePersonalDetailsVerified,
+} from './utils';
+import './VerifyPersonalDetails.scss';
+
+type TVerifyPersonalDetailsProps = {
+    error?: TSocketError<'get_settings'>['error'] | TSocketError<'set_settings'>['error'];
+    onVerification?: VoidFunction;
+};
+
+const VerifyPersonalDetails: React.FC<TVerifyPersonalDetailsProps> = ({ error, onVerification }) => {
+    const { localize } = useTranslations();
+    const { errors, setFieldValue, values } = useFormikContext<TVerifyPersonalDetailsValues>();
+
+    const dateDisplayFormat = 'DD-MM-YYYY';
+
+    const isValid = !errors.dateOfBirth && !errors.firstName && !errors.lastName;
+
+    useEffect(() => {
+        if (values.arePersonalDetailsVerified && onVerification) {
+            onVerification();
+        }
+    }, [onVerification, values.arePersonalDetailsVerified]);
+
+    const handleTNCChecked = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setFieldValue('arePersonalDetailsVerified', event.target.checked);
+    };
+
+    return (
+        <div className='wallets-verify-personal-details'>
+            <InlineMessage>
+                <WalletText size='sm'>
+                    <Localize
+                        components={[<strong key={0} />]}
+                        i18n_default_text='To avoid delays, enter your <0>name</0> and <0>date of birth</0> exactly as it appears on your identity document.'
+                    />
+                </WalletText>
```

Comment:
Since the translations are done in the same PR don't forget to replace the `WalletText` with the deriv-com/ui `Text` component as well to avoid double work for QA afterwards 🙏

Reply:
Hi @lubega-deriv,
Due to time constraints, I will be covering this in a separate task under my epic.